opslevel/resource_opslevel_tag.go

```diff
@@ -0,0 +1,162 @@
+package opslevel
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/opslevel/opslevel-go/v2023"
+)
+
+func resourceTag() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manages a tag. Only uses API's 'tag create', not 'tag assign'.",
+		Create:      wrap(resourceTagCreate),
+		Read:        wrap(resourceTagRead),
+		Update:      wrap(resourceTagUpdate),
+		Delete:      wrap(resourceTagDelete),
+		Schema: map[string]*schema.Schema{
+			"last_updated": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"resource_id": {
```

Comment:
since this takes an alias or an id - would `resource_identifier` be more apt of a name?

Reply:
I think so 👍